src/main/java/frc/robot/subsystems/superstructure/Flywheel.java

```diff
@@ -0,0 +1,108 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems.superstructure;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.ctre.phoenix.sensors.CANCoder;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.PIDSubsystem;
+import edu.wpi.first.wpilibj.controller.SimpleMotorFeedforward;
+import frc.robot.Constants;
+import frc.robot.RobotContainer;
+
+public class Flywheel extends PIDSubsystem {
+
+	// define variables
+	private final WPI_TalonSRX flywheelMain;
+	private final WPI_VictorSPX flywheelSecondary;
+	private final SimpleMotorFeedforward flywheelFeedforward;
+	private CANCoder encoderMain;
```

Comment:
Might as well set this as `final` as well?
```suggestion
	private final CANCoder encoderMain;
```